package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Aug 26 13:18:39 UTC 2022 - Josef Reidinger <jreidinger@suse.com>
+
+- Inform user when running on transactional system that it needs
```

Comment:
What about "On transactional systems, inform the user that packages are required to be installed manually"?